recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ build:
     # cuQuantum only supports Volta+
     - export AER_CUDA_ARCH="7.0;7.5;8.0;8.6;9.0+PTX"  # [cuda_compiler != "None"]
     - CMAKE_GENERATOR="Ninja" {{ PYTHON }} -m pip install . -vv  # [not win]
-  skip: true  # [(win64 and cuda_compiler != "None") or cuda_compiler_version not in ("None", "11.8", "12.5")]
+  skip: true  # [(win64 and cuda_compiler != "None") or cuda_compiler_version not in ("None", "11.8", "12.6")]
```

Comment:
@leofang What is the `cuda_compiler_version not in ("None", "11.8", "12.6")` guarding against? If conda-forge is pinning the cuda version more aggressivley now, maybe we don't need that? It seems like it will mess up the version 12 build again when the pin moves to 12.7.

Reply:
@leofang I rebuilt an environment and realized that qiskit-aer was the only package holding me back from Python 3.13, so I decided to merge this now. I am still curious to hear what you think about the `cuda_compiler_version` guard and can follow up on that in another PR if you think we could remove that guard.

Reply:
Yeah I think it's a legit question. I think it's a limitation today AFAIK of how we express "give me the latest CUDA 11/12/both compiler", but I could be wrong. We can try removing it and see what happens.

Reply:
I tried removing the conditional from the selector locally and the rerender didn't change anything. I will try opening a PR.